<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudents</class>
 <widget class="QDialog" name="AddStudents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Přidat studenty</string>
  </property>
  <widget class="QRadioButton" name="radioButton_Dohromady">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dohromady</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Zvlast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zvlášť</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Soubor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ze souboru</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_First">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>První část</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Second">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Druhá část</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_DohromadyPath1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_DohromadyName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_DohromadyPath2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_DohromadyName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Jméno</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_ZvlastName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_ZvlastPath">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ZvlastName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Jméno:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ZvlastPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cesta:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_SouborPath">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_SouborPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cesta:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DohromadyPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cesta:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Přidat studenta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ButtonAdd">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Write">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit do souboru</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ButtonFile">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
